plugins/terraform/layer0/resource_layer0_load_balancer.go

```diff
@@ -34,6 +34,12 @@ func resourceLayer0LoadBalancer() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"load_balancer_type": {
```

Comment:
[nit] I wonder if "type" is better than the verbose "load_balancer_type" - since we declare it in a `layer0_load_balancer` already, it seems a little much to have to restate that part.

Reply:
I agree that it is verbose. It somewhat stemmed from my implementation of environment types, where type was already taken so I had to create an `environment_type` property. I don't feel strongly either way on this to change it though, so let me know if you consider this is something I definitely need to change and I will update it accordingly.